internal/checkers/contains.go

```diff
@@ -0,0 +1,75 @@
+package checkers
+
+import (
+	"go/ast"
+
+	"golang.org/x/tools/go/analysis"
+)
+
+// Contains detects situations like
+//
+//	assert.True(t, strings.Contains(s, "abc123"))
+//	assert.False(t, strings.Contains(s, "456"))
```

Comment:
Please add the negative

I think people could have used

```go
       assert.True(t, !strings.Contains(s, "abc123"))
```

Bingo https://github.com/search?q=%22assert.True%28t%2C+%21strings.Contains%22&type=code

I didn't find

https://github.com/search?q=%22assert.False%28t%2C+%21strings.Contains%22&type=code

But it might be added maybe, @Antonboom what do you think?

Reply:
You are right

Reply:
To be added to bool-compare, or useless assert maybe @Antonboom may confirm.

I hope he won't say "it's already supported" 😅🤣

Reply:
Found it, bool-compare apparently

https://github.com/Antonboom/testifylint/blob/master/analyzer%2Ftestdata%2Fsrc%2Fbool-compare-custom-types%2Fbool_compare_custom_types_test.go

Reply:
if support of negation is easy, then I allow some linters overlapping
currently some linters saves people from annoying warnings chain

as u wish